tutorials/interoperability/remote-staking-xcm.md

```diff
@@ -29,7 +29,12 @@ For development purposes this tutorial is written for Moonbase Alpha and Moonbas
 
 --8<-- 'text/xcm/calculate-multilocation-derivative-account.md'
 
-The script will return 32-byte and 20-byte addresses. We’re interested in the Ethereum-style account - the 20-byte one. Feel free to look up your multilocation derivative account on [Moonscan](https://moonbase.moonscan.io/){target=_blank}. You’ll note that this account is empty. You’ll now need to fund this account with at least 1.1 DEV. As this is the amount that the faucet dispenses, you'll need to make a minimum of two faucet requests or you can always reach out to us on [Discord](https://discord.com/invite/amTRXQ9ZpW){target=_blank} for additional DEV tokens.
+Here, we have specified a parents value of `1` because the relay chain is the origin of the request (and the relay chain is considered a parent to the Moonbase alpha parachain). The relay chain does not have a parachain id so that field is omitted. 
+
+![Calculate Multi-Location Derivative Account](/images/tutorials/interoperability/remote-staking-via-xcm/xcm-stake-0.png)
```

Comment:
we shouldn't use `0`, instead we should reorganize the images so they always start with `1`. this can be done now (if so, you'll need to also create a CN PR) or at time of translation, but if at time of translation the PR check list will need to be updated. i have no preference as to which way you should go, so i'll leave the decision to you